scripts/build.py

```diff
@@ -1460,51 +1460,30 @@ def shell(self):
 # BEGIN: RELEASE
 #
 
+    def create_tag(self):
+        if self.channel is None:
+            self._log("No channel specified!")
+            sys.exit(1)
 
-    def release(self):
-        page = None;
-        with open(os.path.join("scripts", "resources", "downloads.html"), 'r') as file:
-            page = file.read()
+        is_stable = self.channel == 'stable'
+        channel = '' if is_stable else self.channel
+        msg = 'Release %s%s%s' % (self.version, '' if is_stable else ' - ', channel)
+        tag_name = '%s%s%s' % (self.version, '' if is_stable else '-', channel)
+        cmd = 'git tag -f -a %s -m "%s"' % (tag_name, msg)
 
-        if run.shell_command('git config -l').find('remote.origin.url') != -1 and os.environ.get('GITHUB_WORKFLOW', None) is None:
-            # NOTE: Only run fetch when we have a configured remote branch.
-            # When running on buildbot we don't but fetching should not be required either
-            # as we're already up-to-date
-            self._log('Running git fetch to get latest tags and refs...')
-            run.shell_command('git fetch')
+        # E.g.
+        #   git tag -f -a 1.2.184 -m "Release 1.2.184" <- stable
+        #   git tag -f -a 1.2.184-alpha -m "Release 1.2.184 - alpha"
+        run.shell_command(cmd)
+        return tag_name
 
-        u = urlparse.urlparse(self.get_archive_path())
-        hostname = u.hostname
-        bucket = s3.get_bucket(hostname)
+    def push_tag(self, tag):
+        cmd = 'git push -f origin %s' % tag
+        run.shell_command(cmd)
```

Comment:
We need to push it as well. Using `-f` to update an existing tag (e.g. `1.2.184-alpha`)